prow/github/client.go

```diff
@@ -2793,6 +2794,31 @@ func (c *client) CreateFork(owner, repo string) error {
 	return err
 }
 
+// ListRepoTeams gets a list of all the teams with access to a repository
+// See https://developer.github.com/v3/repos/#list-teams
+func (c *client) ListRepoTeams(org, repo string) ([]Team, error) {
```

Comment:
Can you add new endpoints to `ghmetrics.go` as well?

Reply:
Thanks for the review. `ghproxy` seems to include that path already:
From the code (the very last line is `teams` under /repos/`owner`/`repo`):

https://github.com/kubernetes/test-infra/blob/43ccd36b7941c233db30ed510bf2511fbea98315/ghproxy/ghmetrics/ghpath.go#L30-L83

And from running `ghproxy`:
```
github_request_duration_bucket{path="/repos/:owner/:repo/teams",status="İ",token_hash="...",le="0.025"} 0
github_request_duration_bucket{path="/repos/:owner/:repo/teams",status="İ",token_hash="...",le="0.05"} 0
github_request_duration_bucket{path="/repos/:owner/:repo/teams",status="İ",token_hash="...",le="0.1"} 1
```